cmd/osbuild-worker/main.go

```diff
@@ -90,7 +90,7 @@ func RequestAndRunJob(client *worker.Client, acceptedJobTypes []string, jobImpls
 	logrus.Info("Waiting for a new job...")
 	job, err := client.RequestJob(acceptedJobTypes, common.CurrentArch())
 	if err == worker.ErrClientRequestJobTimeout {
-		logrus.Debugf("Requesting job timed out: %v", err)
+		logrus.Infof("Requesting job timed out: %v", err)
```

Comment:
This is actually expected behaviour, but it's ok to have it here.

Reply:
Hmm, isn't this going to spam journal? If it is, I suggest reverting this change.

Reply:
every 20s yea, and you're right we don't have great filtering mechanisms yet.

Reply:
@diaasami WHile you are on it, can you also change https://github.com/osbuild/osbuild-composer/blob/ab3712fd928d07c9f40651e253262494ae19d843/cmd/osbuild-worker/main.go#L90 to debug? This message is like 99,9% of our logs and it actually isn't that interesting.